packages/lib/src/test/jsonld.test.ts

```diff
@@ -54,6 +54,26 @@ describe('check', () => {
       'Invalid JSON-LD syntax; @context must be an object.'
     );
   });
+
+  it('should return false if there are dropped terms in a nested json ld', async () => {
+    const jsonLD = {
```

Comment:
should be in a fixture

Reply:
A json file checked into source.

Reply:
Should be fixed, json files referenced in unit test from __fixtures__